docs/en/setup/backend/ui-setup.md

```diff
@@ -23,7 +23,7 @@ zipkinServices: ${SW_ZIPKIN_ADDRESS:http://localhost:9412}
 Start a container to connect OAP server whose address is `http://oap:12800`.
 
 ```shell
-docker run --name oap --restart always -d -e SW_OAP_ADDRESS=http://oap:12800 -e SW_ZIPKIN_ADDRESS=http://oap:9412 apache/skywalking-ui:8.8.0
+docker run --name oap --restart always -d -e SW_OAP_ADDRESS=http://oap:12800 -e SW_ZIPKIN_ADDRESS=http://oap:9412 apache/skywalking-ui:8.8.1
```

Comment:
```suggestion
docker run --name oap --restart always -d -e SW_OAP_ADDRESS=http://oap:12800 -e SW_ZIPKIN_ADDRESS=http://oap:9412 apache/skywalking-ui:x.y.z
```

UI version should be aligned with backend OAP version.

Reply:
How about giving `9.0.0` which is the backend version? See https://github.com/apache/skywalking/blob/8763359fc1f4d8111072502bb5c861fb61be1f1a/docs/en/setup/backend/backend-docker.md

Reply:
9.0.0 is nearly one year ago too.
That is why I am asking for x.y.z generally.
We can't remember to update this version by version
Or you could add one line of export a variable, then use it in the docker command.

Reply:
Something like below?

```shell
export skui=9.0.0
docker run --name oap --restart always -d -e SW_OAP_ADDRESS=http://oap:12800 -e SW_ZIPKIN_ADDRESS=http://oap:9412 apache/skywalking-ui:$skui
```

Reply:
Yes, still, don't use 9.0.0. Use x.y.z to indicate a version number.

Reply:
Or, keep the command the same. But add a comment ahead. For instance:

```shell
# The UI image version x.y.z should be the same with the backend
docker run --name oap --restart always -d -e SW_OAP_ADDRESS=http://oap:12800 -e SW_ZIPKIN_ADDRESS=http://oap:9412 apache/skywalking-ui:x.y.z
```

For the new users like me, don't know the rules of the version.

Reply:
The reason of exporting is to make more clear.

If yoi don't know the version, what is point of using SkyWalking?

Reply:
Now, I don't have any doubts about it. Keeping the current status is ok for me. If no one has questions about the document, then I can close this PR.

Reply:
I mean it should be fixed, just don't put 9.0.0 there.

Reply:
> Something like below?
> 
> 
> 
> ```shell
> 
> export version=9.0.0
> 
> docker run --name oap --restart always -d -e SW_OAP_ADDRESS=http://oap:12800 -e SW_ZIPKIN_ADDRESS=http://oap:9412 apache/skywalking-ui:$version
> 
> ```

Using this fix should be good.